tests/console/yast2_cmdline.pm

```diff
@@ -0,0 +1,21 @@
+use base "console_yasttest";
+use testapi;
+
+sub run() {
+    my $self = shift;
+
+    become_root();
+    # Install git
+    assert_script_run 'zypper -n install git-core';
+
+    # Run all the existing YaST CLI tests
+    $self->run_yast_cli_test('yast-network');
+    assert_script_run 'zypper -n install bind yast2-dns-server';
+    $self->run_yast_cli_test('yast-dns-server');
+
+    # Exit from root
+    script_run "exit";
```

Comment:
I'm for single quotes ( I guess C colleagues would disagree ), since they are negligibly faster. But you still don't avoid double quotes when you want to replace variables in your string. And perl printf is slower than double quotes. :)